lib/Parse/ParseStmt.cpp

```diff
@@ -1857,13 +1857,16 @@ ParserResult<Stmt> Parser::parseStmtForEach(LabeledStmtInfo LabelInfo) {
     consumeToken(tok::code_complete);
   } else {
     Container = parseExprBasic(diag::expected_foreach_container);
+    llvm::errs() << "container:\n";
+    if (!Container.isNull()) {
+      Container.get()->dump();
+    }
+    Status |= Container;
```

Comment:
This is causing dumps when building swift and was probably not intentional.

https://ci.swift.org/job/oss-swift-incremental-RA-osx/133/consoleText